include/ibamr/IBFEMethod.h

```diff
@@ -702,6 +778,12 @@ class IBFEMethod : public IBStrategy
      */
     void writeFEDataToRestartFile(const std::string& restart_dump_dirname, unsigned int time_step_number);
 
+    /*!
+     * Return a pointer to the scratch hierarchy used by this object. See the
+     * main documentation of this class for more information.
+     */
+    SAMRAI::tbox::Pointer<SAMRAI::hier::PatchHierarchy<NDIM> > getScratchHierarchy();
```

Comment:
`hier::BasePatchHierarchy`?

Reply:
We do this in a few places and I don't understand why. What is the advantage of returning a pointer to the base class when we already know it is a derived class?

Reply:
For that day in the distant future when we turn on full support for multiblock domains?

Reply:
Ah. I will make the change.